spec/main-spec.js

```diff
@@ -100,6 +100,7 @@ describe('gotests', () => {
       let text = 'package main' + nl + nl + 'func main()  {' + nl + '}' + nl
       text += 'func ReadConfigFile(filePath string) ([]string, error) {' + nl + '}'
       text += 'func  Strangely_named-Function  ( filePath string ) ( []string,error )  {' + nl + '}'
+      text += 'func(t *T) FuncWithReciever(a int) int {' + nl + '}'
```

Comment:
This just adds some text to test input, to actually test if regexp works for it you need to add `expect(functions).toContain('FuncWithReciever')` somewhere around line 114.